desktop_server/src/com/mygdx/game/server/initialize/StartUnitInitializer.java

```diff
@@ -38,7 +35,7 @@ public void initializeTestUnit() {
     }
     int entityId = componentFactory.createEntityId();
     componentFactory.createCoordinateComponent(new Coordinates(0, 0), entityId);
-    var anyConfig = assets.getGameConfigs().get(UnitConfig.class, random.nextInt(GameConfigs.UNIT_MAX - GameConfigs.UNIT_MIN) + GameConfigs.UNIT_MIN);
+    var anyConfig = assets.getGameConfigs().getAny(UnitConfig.class);
```

Comment:
Tu zmieniłem bo to nie ma sensu, żeby się jakoś starać w tworzenie losowych unitów. Ta zmiana to trochę pozostałość tego, że walczyłem z tym, żeby nie trzeba było używać tych wartości Id bezpośrednio.